project/Component/NefSwiftPlayground/Models/PlaygroundBookTemplate.swift

```diff
@@ -0,0 +1,136 @@
+//  Copyright © 2019 The nef Authors.
+
+import Foundation
+
+enum PlaygroundBookTemplate {
+    
+    // MARK: - Code
+    enum Code {
+        static let header = """
```

Comment:
I guess this is necessary as we cannot bundle plain files and copy their contents. It would be ideal to have this as a Swift file that we can compile and check, instead of a string. Likewise for the rest of string in this file.

Reply:
We have several problems here, it is a macOS project and this code is coding for  `UIKit`, `Playground` - if it starts to change, o we detect some issues related to, we will try to change the approach

Reply:
Good.